tests/configs/parameter_https.cfg

```diff
@@ -73,7 +73,7 @@ SEARCH_FOR_CHANNEL: False
 #-- Path to Pour Points File (char) --#
 # A comma separated file of outlets to route to [lons, lats] - one coordinate pair per line (order not important)
 # May optionally include a column [names] - which will (if not aggregating) be included in param file
-FILE_NAME: https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/RVIC/sample_pour.txt
+FILE_NAME: https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/fileServer/datasets/RVIC/sample_pour.txt
 
```

Comment:
I think the ideal use of `pour` is to get user inputs (lon and lat) and create a `pour.txt` file in the `tmp` directory. It is a variable that needs to be dynamically defined by users. I guess we don't have to worry about that here though. I will write an issue.